asg/src/expression/circuit_access.rs

```diff
@@ -173,6 +173,7 @@ impl<'a> FromAst<'a, leo_ast::CircuitStaticFunctionAccessExpression> for Circuit
 
         if let Some(expected_type) = expected_type {
             return Err(AsgError::unexpected_type(expected_type, "none", &value.span).into());
```

Comment:
I think "none" is the expected type based on the code.

Reply:
Just pushed.